backend/prisma/schema.prisma

```diff
@@ -11,71 +11,118 @@ datasource db {
   relationMode = "prisma"
 }
 
-model School {
-  id       Int       @id @default(autoincrement())
-  name     String
-  domain   String
-  students Student[]
-  users    User[]
-
-  @@map("school")
+//
+//
+// ENUMS
+//
+//
+// FOR USE IN USER, ORGANIZATION, EVENT
+
+// FOR USE IN PARTICIPATION
+enum ParticipationStatus {
```

Comment:
so confused why this is associated to an event, I commented it out @nomanSanjari  on the event model. It should be associated to both an event and a user via a middle table thingie

Reply:
I agree, this field doesn't belong here